docs/BUILD.bazel

```diff
@@ -61,9 +61,12 @@ _READMES = {
         failure_message = "Please run:  bazel run //docs:update",
         file1 = k + ".md",
         file2 = v,
-        # No need to run on more than one platform
-        # Also there's some delta here, probably line endings
-        tags = ["no-bazelci-windows"],
+        tags = [
+            # no need to run on more than one platform
+            "no-bazelci-windows",
+            # diff_test has line endings issues on Windows
```

Comment:
is it diff_test or stardoc? I'm still not clear where this bug actually originates

Reply:
hmm. let me double check

Reply:
checked locally. the only difference is the line endings so I'd call it a diff_test issue since we'd expect windows stardoc to output CRLF right?